include/bm/bluetooth/services/ble_hids.h

```diff
@@ -466,29 +496,114 @@ struct ble_hids_config {
 	 */
 	uint16_t *included_services_array;
 	/**
-	 * @brief Security requirement for HID service Protocol Mode characteristic.
-	 *
-	 * @note Only read and write are used.
-	 */
-	struct ble_hids_char_sec protocol_mode_sec;
-	/**
-	 * @brief Security requirement for HID service Control Point characteristic.
-	 *
-	 * @note Only write is used.
-	 */
-	struct ble_hids_char_sec ctrl_point_sec;
-	/**
-	 * @brief Security requirements for HID Boot Keyboard Input Report characteristic.
-	 */
-	struct ble_hids_char_sec boot_mouse_inp_rep_sec;
-	/**
-	 * @brief Security requirements for HID Boot Keyboard Input Report characteristic.
-	 */
-	struct ble_hids_char_sec boot_kb_inp_rep_sec;
-	/**
-	 * @brief Security requirements for HID Boot Keyboard Output Report characteristic.
+	 * @brief Security configuration.
 	 */
-	struct ble_hids_char_sec boot_kb_outp_rep_sec;
+	struct {
+		/**
+		 * @brief Security requirement for HID information characteristic.
+		 */
+		struct {
+			/**
+			 * @brief Security requirement for reading HID Information characteristic
+			 * value.
+			 */
+			ble_gap_conn_sec_mode_t read;
+		} hid_info;
```

Comment:
Elsewhere there is `_char` as postfix, but we could shorten if we wanted, e.g. `sec_mode.ctrl_point_char.read` -> `sec_mode.ctrl_point.read`.